lib/avalanche.ex

```diff
@@ -227,10 +228,26 @@ defmodule Avalanche do
     end
   end
 
+  @doc """
+  List of available Req request options.
+
+  See `Req.request/1` for more information.
+  """
+  @spec available_req_options :: list(atom())
+  def available_req_options do
```

Comment:
does `Req` expose these? Seems like this could potentially be hard to maintain?

Reply:
just a nit

Reply:
I don't think we can get them dynamically from `Req`. Yes, kinda a pain to maintain.
https://hexdocs.pm/req/Req.html#request/1-options

The list is copied from here: https://github.com/wojtekmach/req/blob/main/lib/req.ex#L93